coreos-base/oem-ec2-compat/files/grub-ec2.cfg

```diff
@@ -4,7 +4,7 @@ set oem_id="ec2"
 
 # Blacklist the Xen framebuffer module so it doesn't get loaded at boot
 # Disable `ens3` style names, so eth0 is used for both ixgbevf or xen.
-set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never clocksource=tsc tsc=reliable"
+set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0 transparent_hugepage=never clocksource=tsc tsc=reliable console=tty1 console=ttyS0 xen_blkfront.max_indirect_segments=256"
```

Comment:
Yes, I wasn't sure if it was recommended *OFF* for those, or if just definitely *ON* for sc1/st1. Any idea?

This option seems really poorly documented and hard to find info for